python/ThirteenTeV/Hadronizer/Hadronizer_TuneCP5_13TeV_EEFilter_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.2017MCTunes.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
```

Comment:
@bendavid on this I am a bit more worried: where was this coming from for CUETP8M1 and so forth?!?
Couldn't find any trace in https://github.com/cms-sw/cmssw/tree/CMSSW_9_3_0/Configuration/Generator/python do you know by heart?

Reply:
fixed in cms-sw/cmssw#20571 cms-sw/cmssw#20572